src/modules/dashboard/components/ActionsPage/CountDownTimer/CountDownTimer.tsx

```diff
@@ -143,6 +143,12 @@ const CountDownTimer = ({
     walletAddress,
   ]);
 
+  useEffect(() => {
+    if (state && data) {
+      refetch();
+    }
+  }, [state]);
```

Comment:
So this is technically working, however, when the state changes, there is a brief flash where the time is still shows wrong (adding staking + voting time)

Here's an example:
![bug-vr-counter-wrong-time-flash](https://user-images.githubusercontent.com/1193222/119150528-350e9000-ba57-11eb-94f8-14936f66b0e4.gif)

In my example, the timeout is set to 1h. As you can see when the state changes to Reveal, for a brief second it shows as 2h (voting time + reveal time), then, after the `refetch()` call finishes, it shows the proper time.